### PR TITLE
CoffeeToWaterRatio: Update to full template

### PR DESCRIPTION
**Pull request by lamanh**
Updates CoffeeToWaterRatio Goodie Re: #2767

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @nickselpa
https://duck.co/ia/view/coffee_to_water_ratio

### CODE REVIEW

**Comment by daxtheduck** [comment]
@lamanh Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/CoffeeToWaterRatio.pm [hunk 1]
```diff
@@ -47,18 +47,26 @@ sub convertResult {
...
 
 handle remainder => sub {
 
     return "1 g to 16.7 ml (0.035 oz. to 0.56 fl. oz.)", structured_answer => {
-        input     => [''],
-        operation => 'Coffee to water ratio per gram (0.035 ounces)',
-        result    => "16.7 ml (0.56 fl. oz.)",
+        data => {
+            title    => '16.7 ml (0.56 fl. oz.)',
+            subtitle => 'Coffee to water ratio per gram (0.035 ounces):',
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/CoffeeToWaterRatio.pm]: Lets remove the trailing ':'.

File: t/CoffeeToWaterRatio.t [hunk 2]
```diff
@@ -8,75 +8,45 @@ use DDG::Test::Goodie;
 zci answer_type => "coffee_to_water_ratio";
 zci is_cached   => 1;
 
+sub build_structured_answer {
+    my ($input, $result) = @_;
+    my ($title, $subtitle);
+    if ($input eq '') {
+        $subtitle = "Coffee to water ratio per gram (0.035 ounces):";
```
Comment (GuiltyDolphin) [inline t/CoffeeToWaterRatio.t]: Lets remove the trailing ':'.

**Comment by GuiltyDolphin** [comment]
@lamanh Thanks for this I've left a couple of comments for you.

**Comment by mintsoft** [comment]
@lamanh Just catching up, are you still interested in finishing this off?

**Comment by GuiltyDolphin** [comment]
Leee ping @lamanh how's this going?

**Comment by mintsoft** [comment]
@GuiltyDolphin I'll rescue this at lunch tomorrow if we don't hear back

**Comment by GuiltyDolphin** [comment]
@mintsoft Okay!

tazzina_di_caffe_a_ventimiglia

/cc @tagawa

**Comment by mintsoft** [comment]
Thanks @lamanh I've pulled your changes into : https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3209

**Pull request closed without merging**